src/flo-integration/flo-worker.service.ts

```diff
@@ -19,8 +19,14 @@ export class FloWorkerService {
     private primaryWorker?: FloWorkerInstance;
     private secondaryWorker?: FloWorkerInstance;
     private workers: FloWorkerInstance[] = [];
+    private inited = false;
 
     public initialize() {
+        if (this.inited) {
```

Comment:
Why do we need this?

Reply:
This is because App.vue seems to be recreated when the user switches PTR/Live, causing this function to be called multiple times, resulting in the existence of many flo-worker instances:
https://github.com/w3champions/launcher/blob/china-optimization/src/App.vue#L60
It was indeed a hack, but I couldn't think of a better way to do it for a while.